### ISSUE TITLE
I wish to learn how an actor (process) is created ? [by sending meta-message to its scheduler ?]

### ISSUE THREAD

**Issue by xieyuheng** [kvakvs/beam-wisdoms#3]
I am five.

**Comment by kvakvs** [comment]
A nice question to add to the ELI5 section. I'll begin here and will update the website later.

When you call `spawn` or `spawn_link`, the VM allocates a small piece of memory with a new process header (approx 200-300 bytes) and the a heap with a stack (they live together and take about 1kb). A new process is granted a new unique PID, which won't repeat for a fair amount of days or months after its death, and is registered in a process lookup table.

So the newly created process is not known to any scheduler yet. It is then placed into a run queue for its priority (99.99% of processes run as `normal`), where one of schedulers is able to pick it up and begin executing. Newer versions of Erlang have run queues one per scheduler, but the principle is the same.
